packages/pigment-css-react/tests/createExtendSxProp/runtime-createExtendSxProp.test.js

```diff
@@ -1,11 +1,10 @@
-import { describe } from 'yargs';
 import { expect } from 'chai';
 import createExtendSxProp from '../../src/createExtendSxProp';
 
 describe('createExtendSxProp', () => {
   it('return the new copy of input', () => {
     const original = { color: 'red' };
-    expect(createExtendSxProp()(original)).to.not.equal(original);
+    expect(createExtendSxProp()(original)).not.to.equal(original);
```

Comment:
Nice, fixed in the whole codebase: https://github.com/mui/material-ui/commit/6fe271e65cddd1a33c5539b7b50f279fd14fd6a1, and in MUI X.